src/model/presentation/xml/SVGFile.java

```diff
@@ -219,4 +217,9 @@ public SVGFile clone(){
 		
 	}
 
+	@Override
+	public String getPencilXML(Pencil p) {
```

Comment:
I still think that putting the pencil here rather that like before in a separate class is not necessary, and even a little odd...